component/index.js

```diff
@@ -41,10 +41,10 @@ module.exports = BaseGenerator.extend({
       when: !this.options.name
     }).then(function(first) {
       var name = this.options.name = this.options.name || first.name;
+      name = this.options.name = name.replace(/\/$/, "");
```

Comment:
This could be combined with the above line: `var name = this.options.name = (this.options.name || first.name).replace(/\/$/, "");`

…or maybe the first line could be simplified:

```js
var name = this.options.name || first.name;
name = this.options.name = name.replace(/\/$/, "");
```

Reply:
Ok